example/src/main/kotlin/com/example/afterpay/data/CashData.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.afterpay.data
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class CashData(
```

Comment:
Let's use `kotlin-parcelize` to create `parcelables` automatically, and remove the contents of this data class to keep it lean. Docs: https://developer.android.com/kotlin/parcelize

Reply:
Addressed in [f109244](https://github.com/afterpay/sdk-android/pull/184/commits/f109244e8d5449ae8af7cc72c75de3c65b2238bc)